utils/gpg.py

```diff
@@ -4,6 +4,13 @@
 
 
 def gpg_key_valid(public_gpg_key):
+    if ' ' in public_gpg_key.rstrip():
+        return False
+
+    equal_sign_count = public_gpg_key.count('=')
+    if not public_gpg_key.rstrip().endswith('=' * equal_sign_count):
```

Comment:
`rstrip` is not that expensive, but I'd avoid calling it twice.

Reply:
fixed!